src/components/capture/useDiscoverCapture.ts

```diff
@@ -101,13 +102,23 @@ function useDiscoverCapture(
     const messagePrefix = useFormStateStore_messagePrefix();
 
     // Details Form Store
-    const entityName = useDetailsForm_details_entityName();
+    const formEntityName = useDetailsForm_details_entityName();
     const detailsFormsHasErrors = useDetailsForm_errorsExist();
     const imageConnectorId = useDetailsForm_connectorImage_connectorId();
     const imageConnectorTagId = useDetailsForm_connectorImage_id();
     const imagePath = useDetailsForm_connectorImage_imagePath();
     const setDraftedEntityName = useDetailsForm_setDraftedEntityName();
 
+    const connectorDetails = useConnectorWithTagDetail(null, imageConnectorId);
+    const tagDetails = connectorDetails.connectorTags.find(
+        (tag) => tag.id === imageConnectorTagId
+    );
+
+    const entityName =
+        workflow === 'capture_create' && tagDetails
+            ? `${formEntityName}/${tagDetails.image_name.split('/').at(-1)}`
```

Comment:
You should be able to use `stripPathing` for this.

Reply:
Thanks! done